fe1-web/src/features/digital-cash/reducer/DigitalCashReducer.ts

```diff
@@ -113,8 +120,12 @@ const digitalCashSlice = createSlice({
           if (!(pubKeyHash in laoState.transactionsByPubHash)) {
             laoState.transactionsByPubHash[pubKeyHash] = [];
           }
-          laoState.transactionsByPubHash[pubKeyHash].push(transactionHash);
+
+          if (!laoState.transactionsByPubHash[pubKeyHash].includes(transactionHash)) {
+            laoState.transactionsByPubHash[pubKeyHash].push(transactionHash);
+          }
         });
+        console.log(JSON.stringify(laoState.transactionsByPubHash));
```

Comment:
Is this left here on purpose?

Reply:
No it's not, thanks for pointing this out!